nautobot/extras/templates/extras/approvalworkflowstageinstance_retrieve.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load helpers %}
+
+{% block extra_buttons %}
```

Comment:
Should we actually override block `buttons` instead? It feels to me that we shouldn't have the standard edit/delete buttons on this view.